mne_bids_pipeline/_main.py

```diff
@@ -193,22 +193,19 @@ def main():
         # them twice.
         step_modules = [*STEP_MODULES["init"], *step_modules]
 
-    msg = "Welcome aboard the MNE BIDS Pipeline!"
-    logger.info(**gen_log_kwargs(message=msg, emoji="👋", box="╶╴", step=""))
+    logger.title("Welcome aboard the MNE BIDS Pipeline! 👋")
```

Comment:
we already dropped the article everywhere else:
```suggestion
    logger.title("Welcome aboard MNE-BIDS-Pipeline! 👋")
```